src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -173,8 +147,20 @@ public Pod build() {
             //We need to normalize the path or we can end up in really hard to debug issues.
             final String mountPath = substituteEnv(Paths.get(volume.getMountPath()).normalize().toString().replace("\\", "/"));
             if (!volumeMounts.containsKey(mountPath)) {
-                volumeMounts.put(mountPath, new VolumeMountBuilder() //
-                        .withMountPath(mountPath).withName(volumeName).withReadOnly(false).build());
+                VolumeMountBuilder volumeMountBuilder = new VolumeMountBuilder() //
+                        .withMountPath(mountPath)
+                        .withName(volumeName)
+                        .withReadOnly(false);
+
+                final String volumeSubPath = volume.getSubPath();
+                if (StringUtils.isNotBlank(volumeSubPath)) {
+                    // We need to normalize the subPath, or we can end up in really hard to debug issues Just in case.
+                    final String subPath = substituteEnv(Paths.get(volumeSubPath).normalize().toString().replace("\\", "/"));
```

Comment:
Is that normalization really needed? What happens if the agent is on Windows?

Reply:
Hello Viatombe,
I'm answering this because this piece of code is also in my original PR wich was intended only for ConfigMap volumes.
I think the answer applies on both.
To be honest I do not know what will happen in a windows container.
But what I can see is that that normalization is there since 4 years ago. in commit: [43afca4c3209dc6b1d8b768b157ab53cc12da052](https://github.com/mcamposv/kubernetes-plugin/commit/43afca4c3209dc6b1d8b768b157ab53cc12da052) made by Alex Johnson <ajohnson@bombora.com> 
The only modification I did was to surround it with an if to control whether the new parameter is empty or not.
So I guess that piece of code is as secure or insecure as it was before and is out of the intention of this PR to modify/evaluate that.

Reply:
It's fine, I didn't see that the same code was applied to mountPath.